js/components/main.js

```diff
@@ -281,6 +281,16 @@ class Main extends ImmutableComponent {
       windowActions.setDownloadsToolbarVisible(true)
     })
 
+    ipc.on(messages.WINDOW_MAXIMIZED, () => {
```

Comment:
This is great, thanks! I'll update this tonite to help keep things nice and clean :smile: